.connector-store/meta.json

```diff
@@ -6,5 +6,12 @@
     "description": "",
```

Comment:
What is the reason for not having a description and labels for this?

Reply:
Older store does not have any.